src/com/connectsdk/service/DLNAService.java

```diff
@@ -1055,9 +1055,12 @@ public void run() {
 
                 if (serviceList != null) {
                     for (int i = 0; i < serviceList.size(); i++) {
-                        BasicHttpRequest request = new BasicHttpRequest(SUBSCRIBE, serviceList.get(i).eventSubURL);
+                        String eventSubURL = serviceList.get(i).eventSubURL;
+                        if(eventSubURL == null) return;
```

Comment:
I think it's better to have here 
```java
if(eventSubURL == null) continue; 
```
to subscribe all services with not null URL